packages/storefront-sdk/src/utils/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const dataFetchingMethods = [
+	'content',
+	'navigation',
+	'products',
+	'productCollections',
+	'productCollectionEntries',
+	'query',
+	'spaceProperties'
+] as const;
```

Comment:
We probably will need to rethink this once we have a clearer picture of how plugins will work, and whether these methods (other than `.query`) will be made available by a plugin.